packages/nodes-base/credentials/SpotifyOAuth2Api.credentials.ts

```diff
@@ -0,0 +1,53 @@
+import {
+	ICredentialType,
+	NodePropertyTypes,
+} from 'n8n-workflow';
+
+
+export class SpotifyOAuth2Api implements ICredentialType {
+	name = 'spotifyOAuth2Api';
+	extends = [
+		'oAuth2Api',
+	];
+	displayName = 'Spotify OAuth2 API';
+	properties = [
+		{
+			displayName: 'Spotify Server',
+			name: 'server',
+			type: 'hidden' as NodePropertyTypes,
+			default: 'https://api.spotify.com/',
```

Comment:
Usually, good practise to remove the trailing slash.

Reply:
That is what I thought as well, but for some reason that caused a lot of errors.